sklearn/ensemble/gradient_boosting_cv.py

```diff
@@ -0,0 +1,825 @@
+        effectively inspect more than ``max_features`` features. If a list is
+        given, each item in the list will be evaluated.
+
+    max_leaf_nodes : int, list of int, or None, optional (default=None)
+        Grow trees with ``max_leaf_nodes`` in best-first fashion.
+        Best nodes are defined as relative reduction in impurity.
+        If None then unlimited number of leaf nodes.
+        If not None then ``max_depth`` will be ignored.
+        If a list is given, each item in the list will be evaluated.
+
+    init : BaseEstimator, None, list, optional (default=None)
+        An estimator object that is used to compute the initial
+        predictions. ``init`` has to provide ``fit`` and ``predict``.
+        If None it uses ``loss.init_estimator``.
+        If a list is given, each item in the list will be evaluated.
+
+    random_state : int, RandomState instance or None,
+                   list of int or RandomState, optional (default=None)
+        If int, random_state is the seed used by the random number generator;
+        If RandomState instance, random_state is the random number generator;
+        If None, the random number generator is the RandomState instance used
+        by ``np.random``.
+
+    presort : bool or 'auto', list of string or bool optional (default='auto')
+        Whether to presort the data to speed up the finding of best splits in
+        fitting. Auto mode by default will use presorting on dense data and
+        default to normal sorting on sparse data. Setting presort to true on
+        sparse data will raise an error. If a list is given, each item in the
+        list will be evaluated.
+
+    Attributes
+    ----------
+    grid_scores_ : list of named tuples
+        Contains scores for all parameter combinations in param_grid.
+        Each entry corresponds to one parameter setting.
+        Each named tuple has the attributes:
+
+        - ``parameters``, a dict of parameter settings
+        - ``mean_validation_score``, the mean score over the
+          cross-validation folds
+        - ``cv_validation_scores``, the list of scores for each fold
+
+    best_score_tuple_ : named tuple
+        The attributes of the model which gave the best score.
+
+        - ``parameters``, a dict of parameter settings, which includes the
+          ``n_estimators`` parameter
+        - ``mean_validation_score``, the mean score over the
+          cross-validation folds
+        - ``cv_validation_scores``, the list of scores for each fold
+
+    best_params_ : dict
+        The parameters which gave the best score, along with the
+        ``n_estimators`` parameter.
+
+    best_estimator_ : GradientBoostingClassifier
+        If ``refit`` was ``True``, the model with the best score fit to the
+        entire dataset.
+
+    """
+
+    _estimator_class = GradientBoostingClassifier
+
+    def __init__(self, n_iter_no_change=10, score_precision=2,
```

Comment:
What about the `min_impurity_split` and `criterion` parameters available in `GradientBoostingClassifier`?